azure/template.json

```diff
@@ -244,6 +256,14 @@
                 "name": "WEBSITE_SLOT_POLL_WORKER_FOR_CHANGE_NOTIFICATION",
                 "value": "0"
               }
+              {
+                "name": "SETTINGS__LOGSTASH__HOST",
+                "value": "[parameters('logstashHost')]"
```

Comment:
```suggestion
                "secureValue": "[parameters('logstashHost')]"
```

Reply:
Is this needed for app servers too? For some reason I thought it was only container instances.